### PR TITLE
Release 1.8

### PR DESCRIPTION
**Pull request by mboersma**
Needs proofreading and any suggestions you have to jazz it up a bit. :saxophone:

### CODE REVIEW

File: _posts/2015-07-13-deis-1-8-ceph-optional.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+## What is Deis?
+
+Deis is an open source PaaS that makes it easy to deploy and manage applications on your own servers. Deis builds upon [Docker](http://docker.io/) and [CoreOS](https://coreos.com/) to provide a lightweight PaaS with a Heroku-inspired workflow.
+
+## 1.8.0 Summary
+
+### New Features
+
+- Deis can be provisioned  [without Ceph](http://docs.deis.io/en/latest/managing_deis/running-deis-without-ceph/)
```
Comment (bacongobbler) [inline _posts/2015-07-13-deis-1-8-ceph-optional.md]: accidental double space between "provisioned" and "without"

File: _posts/2015-07-13-deis-1-8-ceph-optional.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+- deis-router uses new nginx `reuseport` [socket sharding](https://www.nginx.com/blog/socket-sharding-nginx-release-1-9-1/)
+- Deis can be provisioned [on Linode](http://docs.deis.io/en/latest/installing_deis/linode/), thanks to community support!
+- `deisctl ssh <component>` opens a shell directly to the node running the specified Deis component
+- deis-router allows a [Diffie-Hellman parameter](https://github.com/deis/deis/pull/3872) for DHE ciphersuites
+- deis-router allows customizing [preferred SSL ciphers](https://github.com/deis/deis/pull/3873)
+- deis-router [enables HSTS](https://github.com/deis/deis/pull/3866) when `enforceHTTPS` is set
```
Comment (bacongobbler) [inline _posts/2015-07-13-deis-1-8-ceph-optional.md]: This is a bit confusing. I believe the workflow is

 - enabled HSTS? enable HTTPS
 - enabled HTTPS? do nothing

This is based off my reading from https://github.com/deis/deis/pull/3866#issuecomment-112196402. Should we change this to "deis-router allows enabling HSTS when `enforceHTTPS` is set"?

File: _posts/2015-07-13-deis-1-8-ceph-optional.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+- Fixed JSON decoding errors in test suite due to premature SSH close
+- deis-builder generates SSH keys on boot when needed
+- Doc updates to upgrade procedure, EC2 PROXY protocol, and cluster sizing for etcd
+- Contrib README.md mentions new community tools for monitoring and backup/restore
+- Added Engine Yard provisioning and Deis Pro to docs
+- `deisctl` won't hang forever waiting for a missing service
+- deis-builder only builds slugrunner/slugbuilder if they're missing
+- Amazon EC2 m4 instance types are now supported in provisioning
+- DigitalOcean support added a couple of eligible regions
+- `DEIS_DRINK_OF_CHOICE=slurm` won't break integration tests now
```
Comment (bacongobbler) [inline _posts/2015-07-13-deis-1-8-ceph-optional.md]: slurm

File: _posts/2015-07-13-deis-1-8-ceph-optional.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title: "Deis v1.8 - Ceph Optional"
```
Comment (carmstrong) [inline _posts/2015-07-13-deis-1-8-ceph-optional.md]: I would call this "Optional Ceph". Seems to read slightly better in my opinion, and that's what we've been calling it elsewhere.

File: _posts/2015-07-13-deis-1-8-ceph-optional.md [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+author: mboersma
+meta:
+  - name: description
+    content: Deis release v1.8
+  - name: keywords
+    content: deis, release, v1.8, stable, production, production-ready, paas, private paas, heroku, github, docker, coreos, enterprise
+---
+
+The Deis project is proud to announce [v1.8.0](https://github.com/deis/deis/releases/tag/v1.8.0), which makes installation of the Ceph components optional, adds `deisctl ssh` and `deis auth:regenerate` commands, and unleashes Deis onto Linode.
```
Comment (carmstrong) [inline _posts/2015-07-13-deis-1-8-ceph-optional.md]: "which makes installation of `deis-store`, `database`, and `logger` optional."? We should probably link to the documentation again here, as people are likely to read that and wonder what it's all about.

File: _posts/2015-07-13-deis-1-8-ceph-optional.md (hunk 1 above)
Comment (carmstrong) [inline _posts/2015-07-13-deis-1-8-ceph-optional.md]: "without Ceph by using external backing stores"?

File: _posts/2015-07-13-deis-1-8-ceph-optional.md (hunk 2 above)
Comment (carmstrong) [inline _posts/2015-07-13-deis-1-8-ceph-optional.md]: As far as I'm concerned, we can just say "Improved security of `deis-router`" and link to all three PRs. Users will probably know more than we do what changed :)

File: _posts/2015-07-13-deis-1-8-ceph-optional.md [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+- deis-router allows a [Diffie-Hellman parameter](https://github.com/deis/deis/pull/3872) for DHE ciphersuites
+- deis-router allows customizing [preferred SSL ciphers](https://github.com/deis/deis/pull/3873)
+- deis-router [enables HSTS](https://github.com/deis/deis/pull/3866) when `enforceHTTPS` is set
+- `deis auth:regenerate` creates a new authentication token
+
+### Improvements
+
+- Source code for `deisctl` is substantially refactored, with beefier, parallelized test coverage
+- Numerous enhancements to API reference documentation and generated code docs
+- deis-controller will properly roll back a release if some containers weren't created
```
Comment (carmstrong) [inline _posts/2015-07-13-deis-1-8-ceph-optional.md]: Should we move this up, or even to "new features"? Or is it not as big of an improvement as I'm thinking?

File: _posts/2015-07-13-deis-1-8-ceph-optional.md [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+
+- deis-router updated nginx to [1.9.2](http://nginx.org/en/CHANGES)
+- deis-registry incorporates some upstream fixes to the "classic" V1 registry
+- deis-database updated wal-e to 0.8.1
+- deis-controller updated psycopg2 to 2.6.1
+- Several components updated to confd 0.10.0
+
+For more details, please see [CHANGELOG.md](https://github.com/deis/deis/blob/master/CHANGELOG.md).
+
+## What's Next
```
Comment (carmstrong) [inline _posts/2015-07-13-deis-1-8-ceph-optional.md]: As @bacongobbler mentioned in Slack, let's add a paragraph here stating that healthchecks were planned for this release but were pulled to fully bake. "Look forward to zero-downtime deploys next release!"

**Comment by mboersma** [comment]
All comments incorporated, please review again as time permits.

File: _posts/2015-07-13-deis-1-8-ceph-optional.md [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+title: "Deis v1.8 - Optional Ceph"
+author: mboersma
+meta:
+  - name: description
+    content: Deis release v1.8
+  - name: keywords
+    content: deis, release, v1.8, stable, production, production-ready, paas, private paas, heroku, github, docker, coreos, enterprise
+---
+
+The Deis project is proud to announce [v1.8.0](https://github.com/deis/deis/releases/tag/v1.8.0), which makes installation of store, database, and logger [optional](http://docs.deis.io/en/latest/managing_deis/running-deis-without-ceph/), adds `deisctl ssh` and `deis auth:regenerate` commands, and unleashes Deis onto Linode.
```
Comment (carmstrong) [inline _posts/2015-07-13-deis-1-8-ceph-optional.md]: "optional when running external backing stores"? Sorry for the additional clarification, @mboersma

File: _posts/2015-07-13-deis-1-8-ceph-optional.md [hunk 9]
```diff
@@ -0,0 +1,140 @@
...
+- deis-registry incorporates some upstream fixes to the "classic" V1 registry
+- deis-database updated wal-e to 0.8.1
+- deis-controller updated psycopg2 to 2.6.1
+- Several components updated to confd 0.10.0
+
+For more details, please see [CHANGELOG.md](https://github.com/deis/deis/blob/master/CHANGELOG.md).
+
+## What's Next
+
+### Comfigurable Healthchecks
```
Comment (carmstrong) [inline _posts/2015-07-13-deis-1-8-ceph-optional.md]: s/Comfigurable/Configurable/

File: _posts/2015-07-13-deis-1-8-ceph-optional.md (hunk 8 above)
Comment (mboersma) [inline _posts/2015-07-13-deis-1-8-ceph-optional.md]: Not at all--keep the comments coming. Thanks!

**Comment by carmstrong** [comment]
LGTM after my last two comments.

**Comment by bacongobbler** [comment]
LGTM!

**Comment by carmstrong** [comment]
:ship:

**Pull request merged**